images/win/scripts/Installers/Install-Msys2.ps1

```diff
@@ -79,13 +79,12 @@ foreach ($arch in $archs)
 Write-Host "`n$dash Clean packages"
 pacman.exe -Scc --noconfirm
 
-Write-Host "`n$dash Installed msys2 packages"
-pacman.exe -Q | grep -v ^mingw-w64-
+$pkgs = pacman.exe -Q
```

Comment:
Some enhancements such as this one, which don't imply changes in the functionality, might be worth being cherry-picked.

Reply:
@eine thanks a lot for detailed explanation! I'll keep only minor script improvements and remove ucrt installation.